cli/src/dump.rs

```diff
@@ -34,6 +34,19 @@ pub fn handle(
             .collect::<TractResult<Vec<InferenceFact>>>()?;
         crate::utils::check_inferred(&*outputs_facts, &*asserts)?;
     }
+    if let Some(asserts) = &params.assertions.assert_op_count {
+        for (name, expected) in asserts {
+            let count = model
+                .eval_order()
+                .context("Cannot assert op count without an eval order")?
+                .into_iter()
+                .filter(|&i| &*model.node_op(i).name() == &*name)
```

Comment:
I used eval_order to iterate over all the nodes in the model. Is there a better way ?

Reply:
It's probably the best way to go. you could also iterate over all the nodes in the model, it would work too. In most cases it will do the same thing, the only case it is different is when there are dead nodes in the graph. For what we are doing it should not matter too much.

But it misses the nested models ! You also need to iterate on the subgraphs or we will miss all the Scan internal nodes... look for "nested_models" in the cli, there are already a few locations when this is done.